src/scss/_tables.scss

```diff
@@ -4,3 +4,47 @@
   font-family: $font-family-sans-serif;
   font-size: $font-size-base;
 }
+
+@media only screen and (max-width: 767px) {
+  table,
+  thead,
+  tbody,
+  th,
+  td,
+  tr {
+    display: block;
+    width: 100%;
+  }
+
+  thead tr {
```

Comment:
Consider extending `.sr-only`  here, or adding `.sr-only` styles manually, instead of using negative top/left positioning